libraries/AP_Hygrometer/LogStructure.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <AP_Logger/LogStructure.h>
+
+#define LOG_IDS_FROM_HYGROMETER \
+    LOG_HYGRO_MSG
+
+// @LoggerMessage: HYGR
+// @Description: Hygrometer sensor data
+// @Field: TimeUS: Time since system startup
+// @Field: I: Hygrometer sensor instance number
+// @Field: Temp: Current temperature
+// @Field: Humi: Current humidity
```

Comment:
I doubt we're going to struggle for size here; could spell out `Humidity` rather than `Humi` if you liked.

Reply:
> I doubt we're going to struggle for size here; could spell out `Humidity` rather than `Humi` if you liked.

yes, i will change it .